modin/core/storage_formats/base/query_compiler.py

```diff
@@ -169,6 +169,9 @@ def default_to_pandas(self, pandas_op, *args, **kwargs):
             warnings.filterwarnings("ignore", category=FutureWarning)
             result = pandas_op(try_cast_to_pandas(self), *args, **kwargs)
         if isinstance(result, (tuple, list)):
+            if "Series.tolist" in pandas_op.__name__:
```

Comment:
Is everything okay here? We register this method like this.

https://github.com/modin-project/modin/blob/e12b21703494dcbb8f7aa951b40ebe1d033307ba/modin/core/storage_formats/base/query_compiler.py#L476

Reply:
Yes, `to_list` is a alias of `tolist`.